options/flags.go

```diff
@@ -95,6 +99,21 @@ func (o *Options) AddFlags(cmd *cobra.Command) {
 		"repository to check (valid inputs: \"owner/repo\", \"github.com/owner/repo\", \"https://github.com/repo\")",
 	)
 
+	cmd.Flags().StringSliceVar(
+		&o.Repos,
+		FlagRepos,
+		o.Repos,
+		"repositories to check (<repo1>, <repo2>, ...). Each repo must be one of:"+
+			" \"owner/repo\", \"github.com/owner/repo\", \"https://github.com/repo\"",
```

Comment:
nit:

since this is a comma separated list, the space you have in your example would break it: `<repo1>, <repo2>, ...`
This should be `<repo1>,<repo2>, ...`

Reply:
changed